src/os.ts

```diff
@@ -0,0 +1,5 @@
+import * as os from 'os'
+
+export const isWindows10 = process.platform === 'win32' && os.release().split('.')[0] === '10'
+
+export const isRS4OrGreater = isWindows10 && !(os.release().split('.')[1] === '0' && parseInt(os.release().split('.')[2]) < 17134)
```

Comment:
Should `isRS4OrGreater` be renamed to `isRS4OrLess` or should `parseInt(os.release().split('.')[2]) < 17134)` be `parseInt(os.release().split('.')[2]) >= 17134)`. What am I missing?

Reply:
Take a closer look. The negative operator is in front of the corresponding conditional statement.
+Of course, this code works well, but I'd appreciate it if you could modify it for readability.

Reply:
That's completely true.